src/components/executive-education-2u/ExecutiveEducation2UPage.jsx

```diff
@@ -55,25 +59,31 @@ function ExecutiveEducation2UPage() {
       <Helmet>
         <title>{pageTitle}</title>
       </Helmet>
-      <h2>
-        {isLoading ? (
-          <Skeleton containerTestId="loading-skeleton-page-title" />
-        ) : (
-          <>{pageTitle}</>
-        )}
-      </h2>
-      <p>
-        {(isLoading || !contentMetadata) ? (
-          <Skeleton count={3} containerTestId="loading-skeleton-text-blurb" />
-        ) : (
-          <>
-            {enterpriseName} has partnered with edX and GetSmarter to offer you high-quality Executive Education
-            courses. To access <strong>&quot;{contentMetadata.title}&quot;</strong>, you must{' '}
-            <strong>accept</strong> Terms of Service and <strong>provide course enrollment data</strong>.
-          </>
-        )}
-      </p>
-      {!isLoading && <UserEnrollmentForm className="mt-5" />}
+      { pageError && <ExecutiveEducation2UErrorPage failureReason={activeQueryParams.get('failure_reason')} httpReferrer={activeQueryParams.get('http_referrer') || null} />}
+      {pageError ? <></> : (
```

Comment:
nit: you can use !pageError && <>...</>, extra space on line 62 in the beginning.